frameworks/kafka/tests/test_zookeeper_auth.py

```diff
@@ -110,8 +110,11 @@ def zookeeper_server(kerberos):
             "security": {
                 "kerberos": {
                     "enabled": True,
-                    "kdc_host_name": kerberos.get_host(),
-                    "kdc_host_port": int(kerberos.get_port()),
+                    "kdc": {
```

Comment:
Has this just been passing because of the defaults?

Reply:
Yes